utilities/backupable/backupable_db.cc

```diff
@@ -2619,6 +2633,10 @@ Status BackupEngineImpl::BackupMeta::LoadFromFile(
   // Failures handled at the end
   std::string line;
   if (backup_meta_reader->ReadLine(&line)) {
+    if (rate_limiter != nullptr) {
```

Comment:
Seems like it would be easier/cleaner to add rate limiting support to LineFileReader.

Reply:
Yes - added this as a motivation in the task to add support for rate limiting in file system wrapper layer or file system layer.